static/sass/_patterns_p-survey-chart.scss

```diff
@@ -44,6 +44,12 @@
       position: absolute;
       top: 0;
       width: 100%;
+
+      strong {
```

Comment:
We have this utility in Vanilla. Could you remove this style and add `.u-truncate` class to the strong element please.